CONTRIBUTING.md

```diff
@@ -19,7 +19,7 @@ docker run -itv $(pwd):/grpc-gateway -w /grpc-gateway --entrypoint /bin/bash --r
     ghcr.io/grpc-ecosystem/grpc-gateway/build-env:1.19 -c '\
         bazel run :gazelle -- update-repos -from_file=go.mod -to_macro=repositories.bzl%go_repositories && \
         bazel run :gazelle && \
-        bazel run :buildifier'
+        BAZEL_CXXOPTS="-std=c++14" bazel run :buildifier'
```

Comment:
Alternatively, we could put `.bazelrc` in the project root. I found that in [`ci.yml`](https://github.com/grpc-ecosystem/grpc-gateway/blob/dd7c267de09fe2bb44ddd5e8214ae9a46527d6ef/.github/workflows/ci.yml#L79) we are setting exactly the same env.

Reply:
We have various environment specific options like the CPU and memory limiters that shouldn't go in `.bazelrc`, but this option does seem like a reasonable thing to put in a repo wide `.bazelrc`.

Reply:
I would probably drop it in the `.bazelrc`, 

```
build --cxxopt='-std=c++14'
```

but I have a pathological aversion to environment variables.